src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -15,34 +15,45 @@ There are several ways to deal with personally identifiable information (PII). B
 
 ## Masking
 
-Masking replaces the text content with something else. The default masking behavior is to replace each character with a `*`. In this example the relevant HTML code is: `<table class="sentry-mask">...</table>`:
+Masking replaces the text content with something else. The default masking behavior is to replace each character with a `*`. Elements with class name `sentry-mask` or the attribute `data-sentry-mask` will be blocked. In this example the relevant HTML code is: `<table class="sentry-mask">...</table>`:
 ![Masking example](./session-replay-masking.png)
 
 ## Blocking
 
-Blocking replaces the element with a placeholder that has the same dimensions. The recording will show an empty space where the content was. In this example the relevant HTML code is: `<table data-sentry-block>...</table>`:
+Blocking replaces the element with a placeholder that has the same dimensions. The recording will show an empty space where the content was. Elements with class name `sentry-block` or the attribute `data-sentry-block` will be blocked. In this example the relevant HTML code is: `<table data-sentry-block>...</table>`:
 ![Blocking example](./session-replay-blocking.png)
 
 ## Ignoring
 
-Ignoring only applies to form inputs. Events will be ignored on the input element so that the replay doesn't show what occurs inside of the input. In the below example, notice how the results in the table below the input changes, but no text is visible in the input:
+Ignoring only applies to form inputs. Events will be ignored on the input element so that the replay doesn't show what occurs inside of the input. Elements with class name `sentry-ignore` or the attribute `data-sentry-ignore` will have their events be ignored. In the below example, notice how the results in the table below the input changes, but no text is visible in the input:
```

Comment:
```suggestion
Ignoring only applies to form inputs. Events will be ignored on the input element so that the replay doesn't show what occurs inside of the input. Any event on an input element with class name `sentry-ignore` or the attribute `data-sentry-ignore` will be ignored. Notice how the results in the table below show input changes, but no visible text:
```

Let me know if my changes work or if they change the meaning of what we're trying to say too much.